<log>
  <summary_ verdicts='3' failures='2' errors='0'/>
  <section_ name='Test' method='verdict_refute?' type='fail'>
    <section_ name='Args=[true, &quot;Message 0&quot;]'>
      <verdict_ method='verdict_refute?' outcome='failed' id='0' message='Message 0'>
        <actual_ class='TrueClass' value='true'/>
        <exception_ class='Minitest::Assertion' message='Expected true to not be truthy.'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
    <section_ name='Args=[&quot;not_nil&quot;, &quot;Message 1&quot;]'>
      <verdict_ method='verdict_refute?' outcome='failed' id='1' message='Message 1'>
        <actual_ class='String' value='&quot;not_nil&quot;'/>
        <exception_ class='Minitest::Assertion' message='Expected # encoding: UTF-8'>
          <backtrace_>
          </backtrace_>
        </exception_>
      </verdict_>
    </section_>
  </section_>
  <verdict_ method='verdict_assert_equal?' outcome='passed' id='error_count'>
    <expected_ class='Integer' value='0'/>
    <actual_ class='Integer' value='0'/>
  </verdict_>
</log>
